kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -762,6 +772,19 @@ protected String appendExtractionExpression(String type, String name, String id,
             jsonTemplate.append("getMultipleInputData('")
                         .append(id)
                         .append("')");
+        } else if (type.equals("datetime-local")) {
+            if (isShowTime) {
+                jsonTemplate.append("document.getElementById('")
+                        .append(id)
+                        .append("')")
+                        .append(getExtractionValue(jsType));
+            } else {
+                jsonTemplate.append("document.getElementById('")
+                        .append(id)
+                        .append("') ")
+                        .append(getExtractionValue(jsType))
+                        .append(" + 'T00:00'");
```

Comment:
@bxf12315 I'm not sure about this extraction value 'is valid for both sides (BC and Rest call)
What I found testing this code is:
- Read date time with time work fine in both
- Write  date time with time works fine just in BC, in REST fails parsing value
- Read date time without time, only works in BC, in REST shows an empty value
- Write date time without time, in BC fail without error(is not saving the value) and in REST: fails parsing value 
@ChughShilpa,  I'm not sure if its something related with my env, could you mind to confirm that this behavior is only happening to me? Thanks!

Reply:
- Write date time with time works fine just in BC, in REST fails parsing value - This works for me

- Read date time without time, only works in BC, in REST shows an empty value - It doesn't work.. Please check Jan's comment above

- Write date time without time, in BC fail without error(is not saving the value) and in REST: fails parsing value - This works with Rest call and for BC - this option is disabled for me although not sure it seems because of some admin rights

Reply:
I think these cases are covered in our summary doc, there are known limitations of datetime types handling and we'll follow up on those.